src/main/java/org/jabref/gui/journals/AbbreviateAction.java

```diff
@@ -53,12 +62,21 @@ public void run() {
 
                 return false;
             };
+            tasks.add(callable);
+        }
+
+        // Execute the callables and wait for the results.
+        List<Future<Boolean>> futures = JabRefExecutorService.INSTANCE.executeAll(tasks);
 
-            boolean result = JabRefExecutorService.INSTANCE.executeAndWait(callable);
-            if (result) {
-                count++;
+        // Evaluate the results of the callables.
+        long count = futures.stream().filter(future -> {
+            try {
+                return future.get();
+            } catch (InterruptedException | ExecutionException exception) {
+                LOGGER.error("Unable to retrieve value.", exception);
+                return false;
```

Comment:
I would not break the loop here, just continue with the next future. If the `return false;` makes sense here, please add a comment why `false` is returned.

Reply:
the return is for the lambda so everything is fine.